CondCore/PPSTimingCalibrationPlugins/plugins/PPSTimingCalibration_PayloadInspector.cc

```diff
@@ -0,0 +1,64 @@
+#include "CondCore/Utilities/interface/PayloadInspectorModule.h"
+#include "CondCore/Utilities/interface/PayloadInspector.h"
+#include "CondCore/CondDB/interface/Time.h"
+#include "CondCore/CondDB/interface/PayloadReader.h"
+#include "CondFormats/PPSObjects/interface/PPSTimingCalibration.h"
+#include "CondCore/PPSTimingCalibrationPlugins/interface/PPSTimingCalibrationPayloadInspectorHelper.h"
+#include <memory>
+#include <sstream>
+
+#include "TH2D.h"
+#include "TCanvas.h"
+#include "TLine.h"
+#include "TLatex.h"
+
+namespace {
+  using namespace PPSTimingCalibrationPI;
```

Comment:
Namespaces should start with the lowercase character, as specified in https://cms-sw.github.io/cms_coding_rules.html